backend/tests/v5/unit/middleware/dataConverter/inputs/auth.test.js

```diff
@@ -0,0 +1,80 @@
+/**
```

Comment:
let's call this file users.test.js (also rename the counter part if its also called auth)